### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by Demo2886**

### CODE REVIEW

File: Alexandr_Nefedin/02.Git.Local/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  165  git log --oneline
+  166  git merge hotfix/we_gonna_die
+  167  git log --oneline
+  168  git checkout master
+  169  histori
+  170  history
+
+```
+
+<h1>master</h1>
```
Comment (pluhin) [inline Alexandr_Nefedin/02.Git.Local/README.md:65]: this is not html, please replace to git markdown

please check whole file

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Alexandr_Nefedin/02.Git.Local/README.md (hunk 1 above)
Comment (Demo2886) [reply Alexandr_Nefedin/02.Git.Local/README.md]: ok, added

**Review by pluhin — APPROVED** [review]
Good, approved

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**